library/gui/tooltip/src/main/java/org/quiltmc/qsl/tooltip/api/package-info.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * <h2>The Quilt Tooltip API</h2>
+ *
+ * <p>
+ * <h3>What are the tooltip APIs?</h3>
+ * With Minecraft 1.17, a new system has been introduced through the {@link net.minecraft.item.BundleItem}: custom tooltip components.
+ * <p>
+ * Those tooltip components are done using mainly two classes:
+ * <ul>
+ *     <li>{@link net.minecraft.client.item.TooltipData} which will hold information about the tooltip, exist on both sides;</li>
```

Comment:
if "exist" refers back to the class, it should be "exists", if it refers back to "tooltip", it should be "that exists"